### PR TITLE
Update Range Slider component to being able to handle units for $, in…

### PR DESCRIPTION
**Pull request by florianzaneray**
…ches and feet. Additionally added ability to add a custom label before and after a unit to increase flexibility.

### CODE REVIEW

File: packages/vue/index/components/base/ZrRangeSlider.vue [hunk 1]
```diff
@@ -103,11 +117,25 @@
         type: Number,
         required: false,
         default: 1
+      },
+      /** Step Size */
```
Comment (cbaxter713) [inline packages/vue/index/components/base/ZrRangeSlider.vue]: Please add a comment above unitType that describes what it is used for, and what values it supports

File: packages/vue/index/components/base/ZrRangeSlider.vue [hunk 2]
```diff
@@ -124,9 +152,51 @@
...
+
+        // format range values depending on unit type
+        switch (this.unitType) {
+
+          case '$':
+            // format range labels as $ currency
+            minValue || minValue === 0 ? minValueDisplay = '$' + minValue.toString().replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,') : ''
+            maxValue ? maxValueDisplay = '$' + maxValue.toString().replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,') : ''
+            break
+          case 'foot-inch-short':
```
Comment (mdmullendore) [inline packages/vue/index/components/base/ZrRangeSlider.vue]: It looks like we can create a helper function for the different inch/foot suffixes; this way we are not repeating the const's.

**Comment by cbaxter713** [comment]
Looks good to me

**Comment by florianzaneray** [comment]
> 🍖

🍰

**Pull request merged**